tensorflow/python/eager/forwardprop.py

```diff
@@ -140,6 +140,40 @@ def _jvp_helper(op_name, attr_tuple, inputs, outputs, tangents):
     return output_tangents
 
 
+def _jvp_helper_wrapper(
+  op_name, attr_tuple, inputs, outputs, tangents, batch_size
+):
+  """Computes a batch of Jacobian-vector product for an op.
+
+  Args:
+    op_name: A string, the type of operation being executed.
+    attr_tuple: Attributes of the operation.
+    inputs: A flat list of input Tensors to the operation.
+    outputs: A flat list of output Tensors from the operation.
+    tangents: A flat list of Tensors, same shape as `[batch_size] + input_shape`.
```

Comment:
nit: missing a batch_size docstring?

And actually Returns: should include [batch_size] too